src/pilot/templates/nic-configs/4_port/nic_environment.yaml

```diff
@@ -26,7 +26,7 @@ parameter_defaults:
   ControllerBond1Interface1: em2
   ControllerBond1Interface2: p1p2
   # The bonding mode to use for controller nodes
```

Comment:
The MTU variables that were added to the NIC configuration files should also be added here.